csrankings.csv

```diff
@@ -8009,6 +8009,7 @@ John Wylie Lloyd,Australian National University,https://cs.anu.edu.au/people/joh
 John Yiannis Cotronis,University of Athens,http://www.di.uoa.gr/eng/staff/1022,9IbOFmMAAAAJ
 John Zahorjan,University of Washington,https://homes.cs.washington.edu/~zahorjan,NOSCHOLARPAGE
 John Zimmerman,Carnegie Mellon University,https://www.cs.cmu.edu/~johnz,A-4JUL4AAAAJ
+John-Paul Ore,North Carolina State University,https//www.csc.ncsu.edu/~jwore/,q7jnx5IAAAAJ

```

Comment:
Typo in the URL, and inserting the : didn't help. Let's just use "https://jpwco.com/".